pkgs/development/interpreters/dhall/default.nix

```diff
@@ -0,0 +1,18 @@
+{ haskell, haskellPackages, stdenv }:
+
+let
+  static = haskell.lib.justStaticExecutables haskellPackages.dhall;
+
+in static.overrideAttrs (old: {
+  passthru = old.passthru or {} // {
+    prelude = stdenv.mkDerivation {
```

Comment:
I think you can use `runCommand` instead of `stdenv.mkDerivation`.  Besides simplifying the expression it also involves an unnecessary build-time dependence on the C compiler:

```nix
prelude = runCommand "dhall-prelude" {} ''
  mkdir $out
  cp -r ${old.src}/Prelude/* $out/
''
```

Reply:
That doesn’t work (tried it) because `old.src` is a `.tar.gz`. What could work is invoking `stdenv` manually, but I don’t know if it’s worth the effort.

Reply:
Done.